Loop/Managers/DeviceAlert/DeviceAlertManager.swift

```diff
@@ -19,24 +20,38 @@ protocol DeviceAlertManagerResponder: class {
 /// - serializing alerts to storage
 /// - etc.
 public final class DeviceAlertManager {
+    static let soundsDirectory = "\(NSSearchPathForDirectoriesInDomains(FileManager.SearchPathDirectory.libraryDirectory, .userDomainMask, true).last!)/Sounds"
+
+    private let log = OSLog(category: "DeviceAlertManager")
```

Comment:
We have a wrapper `DiagnosticLog` you can use here
(forgot to mention this last time I commented about logging)